test/shared/installationVerification.test.ts

```diff
@@ -509,109 +546,6 @@ describe('InstallationVerification Tests', () => {
       });
   });
 
-  it.skip('server error', async () => {
```

Comment:
ditto, this UT was testing the HTTP calls to npm, we run `npm show` now (see prev comment)